notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>maybe put a note that this file is only accessible if you're on the frank lab server that mounts stelmo. others (e.g. from other labs) wouldn't have access to this. might be a good idea to have a dropbox link for this file for such users. in that case, maybe we don't want to use montague, maybe we want to use something that is already public, like the nwb files that Anna uploaded to DANDI.</p><p> </p><p>by the way, the user wouldn't have the _ version nwb file if they didn't run the first tutorial notebook with montague. I think it uses beans right now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Will add that note. I believe eventually this file was created to replace the one currently on DANDI, so that shouldn't be an issue long-term. Happy to make a commit once that happens to include a link. We might think about changing the first tutorial then if we're intent on using a file from DANDI...</p>

Reply:
<p>There may be a good reason not to do this here, but in my opinion, it would be nice for folks to be running this tutorial on a uniquely named copy of the example .nwb instead of all using the exact same .nwb (similar to how I think other tutorials have worked previously; for ex, I would've copied beans.nwb to beans_ac_testing.nwb, then throughout the tutorials would have used the latter file). If we all use the same nwb, then a user isn't forced to run each cell, tables are all pre-populated if anyone else in the lab has previously run the tutorial, and compute times are non-representative because, as it should be, skip_duplicates=True. </p>

Reply:
<p>I think a good way to rectify this might be to have an example_tutorial.nwb that we work from (as to not mess with anyone's original NWB file). Having the user say 'yes' when prompted from <code>SortGroup().set_group_by_shank</code>should also clear any downstream tables. I'll double-check this and maybe add a clean-up line at the end for any tables it misses. </p>

Reply:
Addressing #114 also needs to be a part of using this file.